react/components/RenderProvider.tsx

```diff
@@ -340,15 +338,17 @@ class RenderProvider extends Component<Props, RenderProviderState> {
     const root = page.split('/')[0]
     const editorProvider = extensions[`${root}/__provider`]
     const maybeEditable = !production && editorProvider
-      ? <ExtensionPointComponent component={editorProvider.component} props={{extensions, pages, page}}>{component}</ExtensionPointComponent>
+      ? <ExtensionPointComponent component={editorProvider.component} props={{extensions, pages, page}} treePath="">{component}</ExtensionPointComponent>
```

Comment:
We don't need an `ExtensionPointComponent` here anymore because it doesn't HMR anymore. This can be replaced by a "withHMR" directly.

Reply:
The component is already wrapped by `withHMR`, because it is registered into `__RENDER_7_COMPONENTS__` this way. What `ExtensionPointComponent` is doing here is just grabbing the implementation from this register and rendering it.